proto/Cargo.toml

```diff
@@ -27,6 +27,6 @@ protobuf = "2"
 lazy_static = { version = "1.3", optional = true }
 
 [build-dependencies]
-protobuf-build = { version = "0.11", default-features = false }
-grpcio-compiler = { path = "../compiler", version = "0.5.0", default-features = false }
+protobuf-build = { version = "=0.11.3", default-features = false }
```

Comment:
Why still changing this?

Reply:
protobuf-build must be 0.11.3, or it will produce code with std::future

Reply:
Why? Isn't it constraint by L31?

Reply:
yes, cargo.lock showed that there are 2 grpcio-compiler and I'll patch it later.